bin/prettier.js

```diff
@@ -368,6 +377,24 @@ function writeOutput(result) {
 }
 
 function eachFilename(patterns, callback) {
+  // The ignorer will be used to filter file paths after the glob is checked,
+  // before any files are actually read
+  const ignoreFilePath = options.ignorePath || DEFAULT_IGNORE_FILENAME;
+  const absoluteIgnoreFilePath = path.isAbsolute(ignoreFilePath)
+    ? ignoreFilePath
+    : path.resolve(process.cwd(), ignoreFilePath);
+  let ignoreText = "";
+
+  try {
+    ignoreText = fs.readFileSync(absoluteIgnoreFilePath, "utf8");
+  } catch (readError) {
+    if (readError.code !== FILE_NOT_FOUND_ERROR_CODE) {
+      throw readError;
```

Comment:
Probably do something like this rather than throwing:

```js
console.error(`Unable to read ${ignoreFilePath}: ${readError.toString}`);
process.exit(2);
```